plugins/blip/modalday/ModalDay.js

```diff
@@ -451,9 +457,19 @@ module.exports = {
     };
     _.defaults(opts, defaults);
 
+    var bgDomain = [opts.bgDomain[0]];
+    if (opts.clampTop) {
+      if (opts.bgUnits === 'mg/dL') {
+        bgDomain.push(400);
```

Comment:
Nah just wanted to check at this stage - I would have rounded it up but not for any good reason